images/vscode-ssh-server/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# renovate
+FROM debian:stable-slim
+
+# Install sshd and remove the auto-generated keys
+RUN apt-get update && \
+    apt-get install -y openssh-server curl && \
```

Comment:
It would maybe be a nice feature to run ssh hardening on this during container init, so we'd want to setup ansible here, but it can be a future addition.

Reply:
Yeah, pulling in ansible starts to get this out of lightweight territory. Hardening would be good though.